tests/tdastro/conftest.py

```diff
@@ -6,37 +6,69 @@
 TEST_DIR = os.path.dirname(__file__)
 
 
-@pytest.fixture
+def fixture(func):
```

Comment:
I would prefer to avoid this by either 1) moving the directory information into the base directory or 2) adding the full path to the notebook. See the discussion in the notebook section.